client/cli/src/commands/runcmd.rs

```diff
@@ -559,9 +546,9 @@ fn interface_str(
 		log::warn!("It isn't safe to expose RPC publicly without a proxy server that filters \
 		available set of RPC methods.");
 
-		Ok("0.0.0.0")
+		Ok(Ipv4Addr::UNSPECIFIED.into())
```

Comment:
```suggestion
		Ok(Ipv4Addr::UNSPECIFIED)
```

Reply:
The return type is `IpAddr` (notice lack of protocol version), so I'd have to wrap that as:
```rust
Ok(IpAddr::V4(Ipv4Addr::UNSPECIFIED))
```
so to reduce the stutter I went for the `.into()` approach. Do you want me to use `IpAddr::V4` instead or is `into()` okay?